my-app/src/App.js

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import { CreateReactComponent } from "./home-task-1/create-react-component";
+import { CreateReactPureComponent } from "./home-task-1/create-pure-react-component";
 
 function App() {
+  var pureData = "with data from parent";
+
   return (
     <div className="App">
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
         <p>Hello from Function Component</p>
```

Comment:
It is not Function Component, it is blank message. To understand difference between Functional and Class component please, read this article - https://medium.com/@Zwenza/functional-vs-class-components-in-react-231e3fbd7108

Reply:
I am not able to get your point. why it is not a functional component. Could you please bit elaborate.

Reply:
```<p>Hello from Function Component</p>``` is not functional component, it is JSX tag. Yes, **App** is functional component, but this component is created by default by **create-react-component** package. According to your homework, you shuld create your personal functional component.